packages/sit-onyx/src/components/OnyxBasicDialog/OnyxBasicDialog.vue

```diff
@@ -99,6 +100,7 @@ defineExpose({
       densityClass,
       'onyx-truncation-multiline',
       'onyx-basic-dialog',
+      `onyx-basic-dialog__backdrop--${props.backdrop}`,
```

Comment:
```suggestion
      { 'onyx-basic-dialog--intense': props.backdrop === 'intense' },
```

Since we only need the class for intense backdrop, we could simplify the DOM / class list like this